test/helpers.ts

```diff
@@ -0,0 +1,110 @@
+import * as Web3 from 'web3';
+
+import BigNumber from 'bignumber.js';
+import { assert } from 'chai';
+import { findLast, propEq } from 'ramda';
+import {
+  Method,
+  TransactionLog,
+  TransactionOptions,
+  TransactionResult
+} from 'truffle';
+import { MultiSigWallet } from 'wallet';
+
+declare const web3: Web3;
+
+export async function assertReverts(func: () => void) {
+  try {
+    await func();
+  } catch (error) {
+    assertRevertError(error);
+    return;
+  }
+  assert.fail({}, {}, 'Should have reverted');
+}
+
+export function assertRevertError(error: { message: string }) {
+  if (error && error.message) {
+    if (error.message.search('revert') === -1) {
+      assert.fail(
+        error,
+        {},
+        'Expected revert error, instead got: ' + error.message
+      );
+    }
+  } else {
+    assert.fail(error, {}, 'Expected revert error');
+  }
+}
+
+export function findLastLog(
+  trans: TransactionResult,
+  event: string
+): TransactionLog {
+  return findLast(propEq('event', event))(trans.logs);
+}
+
+export async function getData(func: any, ...args: any[]): Promise<string> {
+  const method = func as Method;
+  const request = await method.request(...args);
+  const [param] = request.params;
+  return param.data;
+}
+
+export function findLastTransactionId(tx: TransactionResult) {
+  const log = findLastLog(tx, 'Submission');
+  assert.isOk(log);
+
+  const paramName = 'transactionId';
+  const transactionId = log.args[paramName] as Web3.AnyNumber;
+  assert.isOk(transactionId);
+
+  return new BigNumber(transactionId);
```

Comment:
No need to cast it to BigNumber. All uint256 numbers returned from Web3 are BigNumbers.